content/contexts/experimenterDetails.tsx

```diff
@@ -0,0 +1,77 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+interface ExperimenterData {
```

Comment:
It *might* be worth it to put these types in `content/types/experimenter.ts`, if they get re-used elsewhere. Though considering they aren't currently exported, maybe that's not a concern.